src/platform/common/platform/fileUtils.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+'use strict';
+
+import { getOSType, OSType } from '../utils/platform';
+import * as path from '../../../platform/vscode-path/path';
+
+export function normCasePath(filePath: string): string {
+    return getOSType() === OSType.Windows ? path.normalize(filePath).toUpperCase() : path.normalize(filePath);
```

Comment:
I think we have similar code already in `fileUtils.node.ts`
Instead of copying it here I think we can move it into a common place and have both use the same shared code.

Reply:
Nice, will remove.